File: Data Structures & Algorithms/missing-number/submission-0.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int res=0;
        for (int i=0;i<=n;i++){
            res = i<n ? res^nums[i] : res;
            res^=i;
        }
        return res;
    }
}

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        stick,move,n=0,0,len(nums)
        while move<n:
            if nums[move]!=val:
                nums[stick],nums[move]=nums[move],nums[stick]
                stick,move=stick+1,move+1
            else:
                move+=1
        return stick

        